docker/php-5.6/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM php:5.6-cli-alpine3.8
+
+ARG DOCKER_USER_ID
+ARG DOCKER_GROUP_ID
+
+RUN if ! getent group "${DOCKER_GROUP_ID}" > /dev/null; \
+    then addgroup -S -g "${DOCKER_GROUP_ID}" devs; \
+  fi \
+  && if ! getent passwd "${DOCKER_USER_ID}" > /dev/null; \
+    then adduser -S -u "${DOCKER_USER_ID}" -G "$(getent group "${DOCKER_GROUP_ID}" | awk -F: '{printf $1}')" dev; \
+  fi \
+  && apk add --no-cache git libxml2-dev openssh-client \
+  && apk add --no-cache --virtual .build-deps autoconf g++ make \
+  # xdebug
+  && pecl install xdebug-2.5.5 \
+  && docker-php-ext-enable xdebug \
+  # composer
+  && curl --output composer-setup.php https://getcomposer.org/installer \
+  && php composer-setup.php --install-dir=/usr/local/bin --filename=composer \
+  && rm composer-setup.php \
+  # xdebug command
+  && curl --location --output /usr/local/bin/xdebug https://github.com/julienfalque/xdebug/releases/download/v1.1.0/xdebug \
```

Comment:
on first look, i thought it's simply fork of the xdebug itself.
maybe consider renaming the cli file?

Reply:
You mean in the upstream repository or here in the Dockerfile?

Reply:
i would dare to suggest both :D xdebug is already a name in ecosystem, reusing it can be confusing